packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -919,6 +924,10 @@ export default {
             }, this.spinnerTimeout);
         },
 
+        setSubmittingState (isFormSubmitting) {
+            this.isFormSubmitting = isFormSubmitting;
+        },
```

Comment:
Nit: probably worth adding a JSDoc here to explain why we're doing this, since I'm quite sure in 2 months someone will come and try to remove this method.